Core/GDCore/IDE/Events/ExpressionValidator.cpp

```diff
@@ -460,6 +460,8 @@ const gd::String& ExpressionValidator::TypeToString(Type type) {
     case Type::NumberOrString:
       return numberOrStringTypeString;
     case Type::Variable:
+    case Type::VariableOrProperty:
```

Comment:
Is there a risk that the caller of this function would expect to get back something like "variableOrProperty" or "variableOrPropertyOrParameter"?